packages/executor/.env.example

```diff
@@ -5,7 +5,5 @@ CHUGSPLASH_EXECUTOR__AMPLITUDE_KEY=<amplitude api key for analytics (optional)>
 CHUGSPLASH_EXECUTOR__LOG_LEVEL=<filter logs based on this level, i.e 'error', 'warning', 'info'>
 IPFS_PROJECT_ID=<ipfs project id to retrieve config file>
 IPFS_API_KEY_SECRET=<ipfs api key to retrieve config file>
-OPT_ETHERSCAN_API_KEY=<Optimism Etherscan API key>
-ETH_ETHERSCAN_API_KEY=<Ethereum Etherscan API key>
-INFURA_API_KEY=<Infura Project API Key>
```

Comment:
Is it okay to remove `INFURA_API_KEY` here? It doesn't look like it's used in the executor package.

Reply:
Yep good catch